CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Added MMDS metrics for `GET`, `PUT` and `PATCH` requests
```

Comment:
I would rephrase to make it clear these are API metrics and not MMDS internal metrics. Maybe smth like:

```
Added API metrics for `GET`, `PUT` and `PATCH` requests on `/mmds` endpoint.
```

Reply:
Also this commit should be squashed with the previous - updating the changelog can go with the code changes.

Reply:
Done